backend/main/community_user_test.go

```diff
@@ -127,6 +127,34 @@ func TestGetCommunityUsersByInvalidType(t *testing.T) {
 	checkResponseCode(t, http.StatusBadRequest, response.Code)
 }
 
+func TestGetCommunityLeaderboard(t *testing.T) {
+	clearTable("communities")
+	clearTable("community_users")
+	clearTable("proposals")
+	clearTable("votes")
+	communityId := otu.AddCommunities(1)[0]
+	proposalIdA := otu.AddActiveProposals(communityId, 1)[0]
+	proposalIdB := otu.AddActiveProposals(communityId, 2)[0]
+	proposalIdC := otu.AddActiveProposals(communityId, 3)[0]
+	voteChoice := "a"
+
+	otu.CreateVoteAPI(proposalIdA, otu.GenerateValidVotePayload("user1", proposalIdA, voteChoice))
```

Comment:
from this line down to line 145.. Perhaps you could move these lines into their own `GenerateValidPayloads` function? And then we would need to call it only once inside this test. You could have params like `count` (how many times its being called) and `users` which could be an array of your user strings? 

We could then reuse it, being able to Generate multiple payloads will be useful for testing!